Sources/GeocodedLocation.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  GeocodedLocation.swift
+//  AddressGeocoder
+//
+//  Created by Htin Linn on 12/15/16.
+//  Copyright © 2016 Prolific Interactive. All rights reserved.
+//
+
+import Foundation
+
+/// Type representing a location coordinate in degrees.
+public typealias LocationDegrees = Double
+
+/// The geocoded location model.
+public struct GeocodedLocation {
+    
+    /// Latitude coordinate of the location.
+    public let latitude: LocationDegrees
```

Comment:
Wouldn't it be better to use a [CLLocationCoordinate2D](https://developer.apple.com/reference/corelocation/cllocationcoordinate2d) type instead?